.github/workflows/docker.yml

```diff
@@ -196,3 +287,49 @@ jobs:
         with:
           category: ${{ matrix.target }}
           sarif_file: "${{ matrix.target }}.sarif"
+
+
+  security-fips:
```

Comment:
That's mostly a duplicate of the security job nope? Could we extract this like the deploy job and reuse it here too?

Reply:
yeah, lets do it